### PR TITLE
[EFM] Renamed EpochCommitSafetyThreshold -> FinalizationSafetyThreshold

### PR DESCRIPTION
**Pull request by durkmurder**

### CODE REVIEW

File: state/protocol/kvstore.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ type KVStoreReader interface {
 	// different extensions can have different view counts.
 	GetEpochExtensionViewCount() uint64
 
-	// GetEpochCommitSafetyThreshold [t] defines a deadline for sealing the EpochCommit
+	// GetFinalizationSafetyThreshold [t] defines a deadline for sealing the EpochCommit
```
Comment (durkmurder) [inline state/protocol/kvstore.go:55]: @AlexHentschel can you I ask you update following comment which explains what is a finalization safety threshold as protocol parameter which is used in several places and not only epoch commitment?

**Review by jordanschalm — APPROVED** [review]

File: cmd/bootstrap/cmd/rootblock.go [hunk 2]
```diff
@@ -138,7 +138,7 @@ func rootBlock(cmd *cobra.Command, args []string) {
 	// validate epoch configs
 	err := validateEpochConfig()
 	if err != nil {
-		log.Fatal().Err(err).Msg("invalid or unsafe epoch commit threshold config")
+		log.Fatal().Err(err).Msg("invalid or unsafe finalization safety threshold config")
```
Comment (AlexHentschel) [inline cmd/bootstrap/cmd/rootblock.go:141]:
```suggestion
		log.Fatal().Err(err).Msg("invalid or unsafe config for finalization safety threshold")
```

File: state/protocol/protocol_state/kvstore/models.go [hunk 3]
```diff
@@ -139,7 +139,7 @@ func (model *Modelv0) GetEpochExtensionViewCount() uint64 {
 	return model.EpochExtensionViewCount
 }
 
-func (model *Modelv0) GetEpochCommitSafetyThreshold() uint64 {
+func (model *Modelv0) GetFinalizationSafetyThreshold() uint64 {
 	return model.EpochCommitSafetyThreshold
```
Comment (AlexHentschel) [inline state/protocol/protocol_state/kvstore/models.go:143]: we are still using the old name here. Its probably a breaking change to rename this value, but maybe we could do that before Crescendo?

File: state/protocol/kvstore.go (hunk 1 above)
Comment (AlexHentschel) [reply state/protocol/kvstore.go:55]: I have updated the documentation of `FinalizationSafetyThreshold` :point_right:  my [PR 6384](https://github.com/onflow/flow-go/pull/6384).

**Review by AlexHentschel — APPROVED** [review]

**Pull request merged**